Makefile

```diff
@@ -94,3 +94,50 @@ rq-requeue-failed: ## Requeue failed RQ jobs
 
 rqinfo: ## See RQ info
 	@$(MAKE) cmd=rqinfo rq-cmd
+
+DEPLOY_STAGING_BRANCH_NAME=heroku
+DEPLOY_STAGING_APP_NAME=openedx-webhooks-staging
+DEPLOY_PROD_APP_NAME=openedx-webhooks
+DEPLOY_STAGING_COMMIT=HEAD
```

Comment:
Nit: Should this be `DEPLOY_STAGING_COMMIT`?

Reply:
It is `DEPLOY_STAGING_COMMIT`.  You mean `DEPLOY_STAGING_BRANCH` maybe?

Reply:
Ugh, yup! Done.